### PR TITLE
o [NEXUS-4754] Don't display validation errors for nexus managed roles

### PR DESCRIPTION
**Pull request by nabcos**

### CODE REVIEW

File: nexus/nexus-webapp/src/main/webapp/js/repoServer/repoServer.RoleEditPanel.js [hunk 1]
```diff
@@ -463,7 +463,11 @@ Ext.extend(Sonatype.repoServer.DefaultRoleEditor, Sonatype.ext.FormPanel, {
           this.find('name', 'id')[0].disable();
           this.find('name', 'name')[0].disable();
           this.find('name', 'description')[0].disable();
-          this.find('name', 'roleManager')[0].disable();
+
+          var roleManager = this.find('name', 'roleManager')[0];
+          roleManager.disable();
+          roleManager.clearValidation();
```
Comment (velo) [inline nexus/nexus-webapp/src/main/webapp/js/repoServer/repoServer.RoleEditPanel.js:469]: Won't this show validation and remove right after? Kinda making the UI to blink?

No idea if that is gonna happen just asking =D

**Comment by velo** [comment]
+1 if UI not blinking
if it is, we may should consider a different approach.

File: nexus/nexus-webapp/src/main/webapp/js/repoServer/repoServer.RoleEditPanel.js (hunk 1 above)
Comment (nabcos) [inline nexus/nexus-webapp/src/main/webapp/js/repoServer/repoServer.RoleEditPanel.js:469]: The change is in the `loadHandler`, so I hope the field is not rendered before ;)
FWIW, I did not notice a flicker.

**Comment by bdemers** [comment]
+1 if no flicker

**Pull request merged**